packages/hub/src/scope.ts

```diff
@@ -65,6 +66,9 @@ export class Scope implements ScopeInterface {
   /** Session */
   protected _session?: Session;
 
+  /** Request Mode Session Status */
+  protected _requestSession?: { status?: RequestSessionStatus };
```

Comment:
We should most likely create a new `RequestSession` type for this, to keep it in line with `Session` type.